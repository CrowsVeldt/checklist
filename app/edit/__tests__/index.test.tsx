import { render } from "@testing-library/react-native";
import EditChecklist from "../index";
import AppProvider from "@/context/AppContext";

describe("<CreateChecklist />", () => {
  test("Index renders correctly", () => {
    // const tree = render(
      // <AppProvider>
        // <EditChecklist />
      // </AppProvider>
    // ).toJSON();

    // expect(tree).toMatchSnapshot();
  });
});
