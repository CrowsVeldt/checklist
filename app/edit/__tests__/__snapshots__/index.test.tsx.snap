// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateChecklist /> Index renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "width": "80%",
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "marginVertical": 10,
        }
      }
    >
      <TextInput
        maxLength={16}
        onChangeText={[Function]}
        style={
          {
            "borderColor": "black",
            "borderRadius": 10,
            "borderWidth": 1,
            "fontSize": 24,
            "width": "100%",
          }
        }
        value=""
      />
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "borderColor": "black",
            "borderWidth": 1,
            "height": 40,
            "justifyContent": "center",
            "marginVertical": 10,
            "width": 40,
          }
        }
      >
        <Text />
      </View>
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "borderColor": "black",
            "borderRadius": 10,
            "borderWidth": 1,
            "padding": 20,
            "width": 140,
          }
        }
      >
        <Text
          style={
            {
              "fontSize": 24,
              "textAlign": "center",
              "textAlignVertical": "center",
            }
          }
        >
          Update
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
